test/user_test.rb

```diff
@@ -0,0 +1,61 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+
+class UserTest < Minitest::Test
+
+  def setup
+    @sal = User.new("Sal")
+    @ali = User.new("Ali")
+    @joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    @joke_2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+  end
+
+  def test_different_comedians_exist
+    assert_instance_of User, @sal
+    assert_instance_of User, @ali
```

Comment:
this second assetion is thorough but not really needed. If it works for Sal, it'll work for any `User.new` called in the same way with a name parameter